src/codecs/raw.js

```diff
@@ -7,11 +7,11 @@ import { codec } from './codec.js'
  * @param {Uint8Array} bytes
  * @returns {Uint8Array}
  */
-const raw = (bytes) => coerce(bytes)
+const rawEncodeDecode = (bytes) => coerce(bytes)
```

Comment:
Seems redundant, `coerce` could be used directly

Reply:
@vmx could please either add comment to why we need to alias `coerce` or just get rid of one ?